src/cdk/drag-drop/directives/drag.spec.ts

```diff
@@ -3771,6 +3825,43 @@ class WrappedDropContainerComponent {
   @Input() items: string[];
 }
 
+@Component({
+  template: `
+    <div style="position: absolute; width: 400px; height: 400px; overflow: hidden;">
```

Comment:
Can you move these styles into the `styles` property? Also the `overflow: hidden` seems unnecessary.